project/ROADMAP.md

```diff
@@ -0,0 +1,5 @@
+Galaxy Project Roadmap
```

Comment:
@martenson Now I'm really confused.  Yes, what I was proposing was that the contents of this file mirror exactly what is in the github roadmap issue.  It's just a stub for now.